sumologic/resource_sumologic_dashboard.go

```diff
@@ -79,7 +79,7 @@ func resourceSumologicDashboard() *schema.Resource {
 			"refresh_interval": {
 				Type:         schema.TypeInt,
 				Optional:     true,
-				ValidateFunc: validation.IntInSlice([]int{120, 300, 900, 1800, 3600, 7200, 86400}),
+				ValidateFunc: validation.IntInSlice([]int{0, 120, 300, 900, 1800, 3600, 7200, 86400}),
```

Comment:
The API has a different set of values https://api.sumologic.com/docs/#operation/createDashboard. Which ones are correct?

Reply:
We enforce these on the backend, should use this set
"2m", "5m", "15m", "30m", "1h", "2h", "1d"

Reply:
Can you approve this and I'll make changes to the dashboard api docs?